src/osd/osd_types.h

```diff
@@ -3290,13 +3290,12 @@ struct pg_log_entry_t {
     MODIFY = 1,   // some unspecified modification (but not *all* modifications)
     CLONE = 2,    // cloned object from head
     DELETE = 3,   // deleted object
-    BACKLOG = 4,  // event invented by generate_backlog [deprecated]
-    LOST_REVERT = 5, // lost new version, revert to an older version.
-    LOST_DELETE = 6, // lost new version, revert to no object (deleted).
-    LOST_MARK = 7,   // lost new version, now EIO
-    PROMOTE = 8,     // promoted object from another tier
-    CLEAN = 9,       // mark an object clean
-    ERROR = 10,      // write that returned an error
+    LOST_REVERT = 4, // lost new version, revert to an older version.
```

Comment:
PR heading spelling of remove is wrong

Reply:
The IDs can't change as they are part of the on-disk format